packages/data/src/data-manager.ts

```diff
@@ -233,10 +233,16 @@ export class DataManager implements DataManagerInterface {
       (id) => String(experience?.id) === String(id)
     );
     // Check environment
-    const isEnvironmentMatch = Array.isArray(experience?.environments)
-      ? !experience.environments.length || // skip if empty
-        experience.environments.includes(environment)
-      : true; // skip if no environments
+    const experienceEnvironment =
```

Comment:
@abbaseya you should update code to only use `environment` field (the plural is there only for compatibility on older versions of the SDK/clients...

Reply:
@clllaur  done